frb_codegen/src/parser/ty.rs

```diff
@@ -89,24 +90,36 @@ impl SupportedInnerType {
     /// or `None` otherwise.
     pub fn try_from_syn_type(ty: &syn::Type) -> Option<Self> {
         match ty {
+            syn::Type::Path(syn::TypePath { path, .. })
+                if path.segments.last().unwrap().ident == "Backtrace" =>
+            {
+                Some(SupportedInnerType::Path(SupportedPathType {
+                    ident: Ident::new("String", path.segments.last().unwrap().ident.span()),
+                    generic: vec![],
+                    is_exception: false,
+                }))
+            }
```

Comment:
It is possible we do not do this special treatment to Backtrace...

For example, we know we have a ton of "delegate" types in our Ir. Is it possible to make Backtrace one of them?

Reply:
`SupportedPathType` has no delegate:

```rust
pub enum SupportedInnerType {
    /// Path types with up to n generic type argument on the final segment. All segments before
    /// the last segment are ignored. The generic type argument must also be a valid
    /// `SupportedInnerType`.
    Path(SupportedPathType),
    /// Array type
    Array(Box<Self>, usize),
    /// The unit type `()`.
    Unit,
}

```
I don't understand

Reply:
Well my question is, when observing a backtrace, we write down (and fool the other sub-systems of this package) `ident: "String"`, so all other subsystems will think we are treating a String. That does sound like a hack.

Is it possible that we write down the correct `ident` (sth like `ident: the_element.ident`)...

If we really need to treat it as if a String, https://github.com/fzyzcjy/flutter_rust_bridge/blob/master/frb_codegen/src/ir/ty_delegate.rs IrTypeDelegate may be helpful.

Reply:
But I must return a `SupportedInnerType`. a `IrTypeDelegate` does not fit into that, so what should I do?

Reply:
Well, maybe we can return normally as if we are dealing with String, Vec etc

Reply:
Sorry I don't understand

Reply:
1. In this code, maybe we can treat Backtrace using the same way as treating String/Vec/...
2. When converting types into IR, maybe we can convert Backtrace to a subtype of IrTypeDelegate if needed